provider/oracle/instance.go

```diff
@@ -204,17 +204,28 @@ func (o *oracleInstance) deleteInstanceAndResources(cleanup bool) error {
 			break
 		}
 
+		//
+		// seclist, vnicset, secrules, and acl created with
+		// StartInstanceParams.InstanceConfig.MachineId,
+		// convert o.Id() to machineId for deletion.
+		// o.Id() returns a string in hostname form.
+		tag, err := o.env.namespace.MachineTag(string(o.Id()))
+		if err != nil {
+			return err
```

Comment:
We want to either Trace() or Annotate()
Probably the latter

Reply:
Added Annotate()